WordPressAuthenticator/Unified Auth/View Related/Reusable Views/TextFieldTableViewCell.swift

```diff
@@ -66,6 +70,22 @@ private extension TextFieldTableViewCell {
 }
 
 
+// MARK: - UITextFieldDelegate conformance
+extension TextFieldTableViewCell: UITextFieldDelegate {
+    func textFieldDidChangeSelection(_ textField: UITextField) {
```

Comment:
Just a thought here: `TextFieldTableViewCell` is the `UITextFieldDelegate`, but the delegate methods are actually executing closures that are defined in `SiteAddressViewController`. What if `SiteAddressViewController` was instead set as the 'UITextFieldDelegate` and implements these methods directly?

Reply:
I was trying not to bloat the ViewController, but I agree. It's easier to understand and easier to use if the textfield delegate methods are the SiteAddressViewController's responsibility.